osbrain/tests/test_bugs.py

```diff
@@ -21,10 +21,6 @@ def inc(agent):
     agent = run_agent('a0')
     agent.set_attr(count=0)
     agent.each(0.0, inc)
-    while True:
-        count = agent.get_attr('count')
-        if count > sys.getrecursionlimit():
-            return
-        time.sleep(0.1)
-        if count == agent.get_attr('count'):
-            raise RuntimeError('Probable recursion level exceeded!')
+
+    limit = sys.getrecursionlimit()
+    assert wait_agent_condition(agent, lambda agent: agent.count > limit)
```

Comment:
Note that before, there was no time limit for this test, but now the time limit is set to 3 seconds by the `wait_agent_condition(...)` call.

In my case, the recursion limit is 1000 but in Windows, it only seems to be able to get to around 700. Perhaps we could increase the timer for this particular test.